src/message_interface/client_method.rs

```diff
@@ -73,6 +73,15 @@ pub enum ClientMethod {
         #[serde(rename = "preparedTransactionData")]
         prepared_transaction_data: PreparedTransactionDataDto,
     },
+    #[cfg(feature = "stronghold")]
+    /// Store a mnemonic in the Stronghold vault
```

Comment:
Nitpicking but can you switch these ? We usually put the doc first then all attributes